resources/views/modal.blade.php

```diff
@@ -100,22 +100,22 @@
                                     </div>
                                 </div>
                             </div>
```

Comment:
What's happen here? For the second form is it not going to hide the section anymore?